### ISSUE TITLE
Async API Integration

### ISSUE THREAD

**Issue by JamRamPage** [ibm-messaging/mq-ansible#19]
Over time, the module should fleshed out to allow greater configuration detail to be expressed in YAML. This should eventually be integrated with the Async API project.

**Comment by martinevansibm** [comment]
I think this is part of a broader design decision around the interface for configuring MQ i.e. do we want a YAML interface or do we stick with MQSC.

**Comment by JamRamPage** [comment]
Yes, this was on the backlog of the internal repo, however as we don't currently have Zenhub here, I'm not sure of the best way to show that.
